test/lit.cfg.py

```diff
@@ -166,7 +166,6 @@ def one_line(s):
         %kore-rich-header %s > %t.header.bin
         for out in %test-dir-out/*.proof.out.diff; do
             in=%test-dir-in/`basename $out .proof.out.diff`.in
-            rm -f %t.out.bin
```

Comment:
So, originally I removed this because it was an incorrect form of the `rm -f $hint` line that I had also added. However, it looks like that line was removed from the diff because when I merged with develop, it picked up the fact that you had independently made that identical change. As a result, the diff now shows only this single line. With that being said, this line was the incorrect form of the new line that you also added and can be removed.